src/server/gc_worker/rawkv_compaction_filter.rs

```diff
@@ -196,6 +196,8 @@ impl RawCompactionFilter {
             if commit_ts.into_inner() >= self.safe_point {
                 return Ok(CompactionFilterDecision::Keep);
             }
+
+            self.versions += 1;
```

Comment:
Should only entries with `!raw_value.is_valid()` be counted ?

Reply:
In txn gc of compaction_filter.rs  records will ```self.versions += 1;```,
which is not match ```commit_ts > self.safe_point || value_type != CompactionFilterValueType::Value```

Reply:
```versions``` means: Total stale versions meet by compaction filters.
It is defined in ```CompactionFilterStats```
```
pub struct CompactionFilterStats {
    pub versions: Cell<usize>, // Total stale versions meet by compaction filters.
    pub filtered: Cell<usize>, // Filtered versions by compaction filters.
    pub last_report: Cell<Instant>,
}
```